packages/authorization-process/test/api/setAdminToClient.test.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import { generateToken, getMockClient } from "pagopa-interop-commons-test";
+import { authRole } from "pagopa-interop-commons";
+import request from "supertest";
+import { generateId, TenantId, UserId } from "pagopa-interop-models";
+import { authorizationApi } from "pagopa-interop-api-clients";
+import { api, authorizationService } from "../vitest.api.setup.js";
+import { clientToApiClient } from "../../src/model/domain/apiConverter.js";
+import {
+  clientAdminAlreadyAssignedToUser,
+  clientKindNotAllowed,
+  clientNotFound,
+  userWithoutSecurityPrivileges,
+} from "../../src/model/domain/errors.js";
+
+describe("API POST /clients/{clientId}/admin test", () => {
+  const mockClient = getMockClient();
+  const body = { adminId: generateId<UserId>() };
+  const makeRequest = async (token: string, clientId: string = mockClient.id) =>
+    request(api)
+      .post(`/clients/${clientId}/admin`)
+      .set("Authorization", `Bearer ${token}`)
+      .set("X-Correlation-Id", generateId())
+      .send(body);
+
+  const apiResponse = authorizationApi.Client.parse(
+    clientToApiClient(mockClient, { showUsers: false })
+  );
+
+  beforeEach(() => {
+    authorizationService.setAdminToClient = vi
+      .fn()
+      .mockResolvedValue({ client: mockClient, showUsers: false });
```

Comment:
```suggestion
      .mockResolvedValue({ client: mockClient, showUsers: true });
```
It doesn't change anything but it should always return `true` for `showUsers`